network-api/networkapi/news/factory.py

```diff
@@ -32,34 +32,15 @@ class Meta:
         exclude = ("headline_sentence",)
 
     class Params:
-        unpublished = Trait(
-            publish_after=Faker("date_time", tzinfo=timezone.utc)
-            if RANDOM_SEED and not TESTING
-            else Faker("future_datetime", end_date="+30d", tzinfo=timezone.utc)
-        )
-        has_expiry = Trait(
-            expires=Faker("date_time", tzinfo=timezone.utc)
-            if RANDOM_SEED and not TESTING
-            else Faker("future_datetime", end_date="+30d", tzinfo=timezone.utc)
-        )
-        expired = Trait(
-            expires=Faker("date_time", tzinfo=timezone.utc)
-            if RANDOM_SEED and not TESTING
-            else Faker("past_datetime", start_date="-30d", tzinfo=timezone.utc)
-        )
         video = Trait(is_video=True)
 
     headline = LazyAttribute(lambda o: o.headline_sentence.rstrip("."))
     outlet = Faker("company")
-    date = Faker("date") if RANDOM_SEED and not TESTING else Faker("past_date", start_date="-30d")
+    date = Faker("past_date", start_date=datetime(2020, 1, 1), tzinfo=timezone.utc)
     link = Faker("url")
     excerpt = Faker("paragraph", nb_sentences=3, variable_nb_sentences=True)
     author = Faker("name")
-    publish_after = (
-        Faker("date_time", tzinfo=timezone.utc)
-        if RANDOM_SEED and not TESTING
-        else Faker("past_datetime", start_date="-30d", tzinfo=timezone.utc)
-    )
+    publish_after = Faker("past_datetime", start_date=datetime(2020, 1, 1), tzinfo=timezone.utc)
```

Comment:
Same question here. I'm curious why RANDOM_SEED and TESTING matter when it comes to generating value for `publish_after`